llvm/docs/CommandGuide/llvm-symbolizer.rst

```diff
@@ -207,6 +207,14 @@ Example 7 - Addresses as symbol names:
   foz
   /tmp/test.h:1:0
 
+Example 8 - Addresses having approximate line info:
+
+.. code-block:: console
+
+  $ llvm-symbolizer --obj=test.elf --approximate-line 0xa
```

Comment:
`--approximate-line` -> `--skip-line-zero`

Reply:
Yeah forgot to edit at this place. Thanks again.